app/src/main/AndroidManifest.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="com.example.molliestephenson.tictactoe">
+          package="com.example.molliestephenson.tictactoe">
```

Comment:
Great work. I think I still see references to `com.example.*` in the new changes of this PR.